src/components/UserNavbarItem/item.desktop.tsx

```diff
@@ -1,43 +1,94 @@
-import Link from '@docusaurus/Link';
+import React from 'react';
 import clsx from 'clsx';
-import React, { useState } from 'react';
-import AccountSwitcher from '../AccountSwitcher';
+import { Button } from '@deriv-com/quill-ui';
+import {
+  LabelPairedGridLgRegularIcon,
+  StandaloneRightFromBracketBoldIcon,
+} from '@deriv/quill-icons';
+import useLogout from '@site/src/hooks/useLogout';
+import useDeviceType from '@site/src/hooks/useDeviceType';
 import { IUserNavbarItemProps } from './item.types';
 import styles from './UserNavbarItem.module.scss';
-import SearchButton from '../SearchButton';
+
+interface IActionProps {
+  handleClick: () => void;
+  isDesktop: boolean;
+}
+
+const DashboardActions: React.FC<IActionProps> = ({ handleClick, isDesktop }) => {
+  return (
+    <React.Fragment>
+      <Button
+        onClick={() => location.assign('/dashboard')}
+        type='button'
+        className={styles.dashboardButton}
+        variant='tertiary'
+        color='black'
+        icon={<LabelPairedGridLgRegularIcon />}
+        data-testid='da_login'
+      >
+        Dashboard
+      </Button>
+      {isDesktop && (
+        <Button
+          onClick={handleClick}
+          type='button'
+          variant='tertiary'
+          color='black'
+          className={styles.logoutButton}
+          icon={<StandaloneRightFromBracketBoldIcon />}
+          data-testid='da_logout'
+        >
+          Log out
+        </Button>
+      )}
+    </React.Fragment>
+  );
+};
+
+const SignedInActions: React.FC<IActionProps> = ({ handleClick, isDesktop }) => {
+  const signedInButtonClasses = clsx('navbar__item', styles.UserNavbarItem, styles.SignedInButton);
+
+  return (
+    <nav className='right-navigation'>
+      <Button
+        variant='secondary'
+        color='black'
+        onClick={handleClick}
+        className={signedInButtonClasses}
+        data-testid='sa_login'
+      >
+        Login
+      </Button>
+      {isDesktop && (
+        <Button
+          variant='primary'
+          onClick={() => location.assign('https://deriv.com/signup/')}
+          className={signedInButtonClasses}
+          data-testid='sa_signup'
+        >
+          Sign up
+        </Button>
+      )}
+    </nav>
+  );
+};
 
 const UserNavbarDesktopItem = ({ authUrl, is_logged_in }: IUserNavbarItemProps) => {
-  const [toggle_search, setToggleSearch] = useState<boolean>(false);
+  const { logout } = useLogout();
+  const { deviceType } = useDeviceType();
+  const isDesktop = deviceType === 'desktop';
+
+  console.log('is_logged_in', is_logged_in);
```

Comment:
Was this meant for debugging, or we need this?

Reply:
Fixed